### PR TITLE
Refactored Login & Signup

### PR DESCRIPTION
**Pull request by MartyTheeMartian**
* No longer directly manipulates the DOM
* `onSubmit()` will clear `this.username` * `this.password` after encryption step and POST request

### CODE REVIEW

File: src/components/NavBar/LogIn/LogIn.js [hunk 1]
```diff
@@ -2,29 +2,43 @@ import React, { Component } from 'react';
...
+    super(props);
 
-  componentDidMount() {
-    let username = localStorage.getItem('username');
-    if(username) {
-      document.getElementById('input-username-login').value = username;
-      document.getElementById('remember-username').checked = 'checked';
-      // Refactor using React's `ref` attribute
+    this.rememberChecked = '';
+    this.usernameRemembered = '';
```
Comment (elimisteve) [inline src/components/NavBar/LogIn/LogIn.js:9]: @MartyTheeMartian How about `rememberUsername`, for consistency?

Comment (MartyTheeMartian) [reply src/components/NavBar/LogIn/LogIn.js:9]: @elimisteve good catch :)

Comment (MartyTheeMartian) [reply src/components/NavBar/LogIn/LogIn.js:9]: @elimisteve Can you merge this PR? Next PR will have that line added.

Comment (elimisteve) [reply src/components/NavBar/LogIn/LogIn.js:9]: Want to force push?

**Pull request closed without merging**